internal/policy/operator/deployable_by_olm.go

```diff
@@ -507,55 +523,57 @@ func (p *DeployableByOlmCheck) installedCSV(ctx context.Context, operatorData op
 }
 
 func (p *DeployableByOlmCheck) cleanUp(ctx context.Context, operatorData operatorData) {
-	log.L().Debug("Dumping data in artifacts/ directory")
+	logger := logr.FromContextOrDiscard(ctx)
+
+	logger.V(1).Info("dumping data in artifacts/ directory")
 
 	subs, err := p.openshiftClient.GetSubscription(ctx, operatorData.App, operatorData.InstallNamespace)
 	if err != nil {
-		log.L().Warn("unable to retrieve the subscription")
+		logger.Info("warning: unable to retrieve the subscription")
 	} else {
 		err := p.writeToFile(ctx, subs)
 		if err != nil {
-			log.L().Errorf("could not write subscription to storage")
+			logger.Error(err, "could not write subscription to storage")
 		}
 	}
 
 	cs, err := p.openshiftClient.GetCatalogSource(ctx, operatorData.App, operatorData.InstallNamespace)
 	if err != nil {
-		log.L().Warn("unable to retrieve the catalogsource")
+		logger.Info("warning: unable to retrieve the catalogsource")
 	} else {
 		if err := p.writeToFile(ctx, cs); err != nil {
-			log.L().Errorf("could not write catalogsource to storage")
+			logger.Error(err, "could not write catalogsource to storage")
 		}
 	}
 
 	og, err := p.openshiftClient.GetOperatorGroup(ctx, operatorData.App, operatorData.InstallNamespace)
 	if err != nil {
-		log.L().Warn("unable to retrieve the operatorgroup")
+		logger.Info("warning: unable to retrieve the operatorgroup")
 	} else {
 		if err := p.writeToFile(ctx, og); err != nil {
-			log.L().Errorf("could not write operatorgroup to storage")
+			logger.Error(err, "could not write operatorgroup to storage")
 		}
 	}
 
 	installNamespace, err := p.openshiftClient.GetNamespace(ctx, operatorData.InstallNamespace)
 	if err != nil {
-		log.L().Warn("unable to retrieve the install namespace")
+		logger.Info("warning: unable to retrieve the install namespace")
 	} else {
 		if err := p.writeToFile(ctx, installNamespace); err != nil {
-			log.L().Errorf("could not write install namespace to storage")
+			logger.Error(err, "could not write install namespace to storage")
 		}
 	}
 
 	targetNamespace, err := p.openshiftClient.GetNamespace(ctx, operatorData.TargetNamespace)
 	if err != nil {
-		log.L().Warn("unable to retrieve the target namespace")
+		logger.Info("warning: unable to retrieve the target namespace")
 	} else {
 		if err := p.writeToFile(ctx, targetNamespace); err != nil {
-			log.L().Errorf("could not write target namespace to storage")
+			logger.Error(err, "could not write target namespace to storage")
 		}
 	}
 
-	log.L().Trace("Deleting the resources created by DeployableByOLM Check")
+	logger.V(2).Info("deleting the resources created by DeployableByOLM Check")
```

Comment:
`V(2)` is Debug? Trace? I somewhat prefer a constant/variable value so I don't have to guess. `DBG` and `TRC`?

Reply:
V(2) is trace.

Yeah... log.DBG or log.DEBUG. I do like the smaller one. I'll add that back